### ISSUE TITLE
pip install librosa fails on llvmlite

### ISSUE THREAD

**Issue by stefan-falk** [librosa/librosa#1104]
#### Description

I am trying to install `librosa` in my conda environment (Python 3.5.6) but I keep failing to do so due to `llvmlite`.

It has to be said that I cam failing to do so on two machines: One is a Ubuntu 16.04 the other is a MacBook on macOS Catalina. On both machines I am having the same issue.

See my [question on stackoverflow](https://stackoverflow.com/questions/61475274/runtimeerror-path-failed-executing-please-point-llvm-config-to-the-path-for) regarding the llvm issue.

I recall installing librosa a while ago where it didn't seem that problematic. 

Anyway, am I missing something here?

#### Environment
```
$ python --version
Python 3.5.6 :: Anaconda, Inc.
$ uname -a
Linux everest10 4.15.0-96-generic #97~16.04.1-Ubuntu SMP Wed Apr 1 03:03:31 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
````

#### Versions

```
$ python
Python 3.5.6 |Anaconda, Inc.| (default, Aug 26 2018, 21:41:56) 
[GCC 7.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import platform; print(platform.platform())
Linux-4.15.0-96-generic-x86_64-with-debian-stretch-sid
>>> import sys; print("Python", sys.version)
Python 3.5.6 |Anaconda, Inc.| (default, Aug 26 2018, 21:41:56) 
[GCC 7.3.0]
>>> import numpy; print("NumPy", numpy.__version__)
NumPy 1.18.3
>>> import scipy; print("SciPy", scipy.__version__)
SciPy 1.4.1
>>> import librosa; print("librosa", librosa.__version__)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named 'librosa'
```

**Comment by bmcfee** [comment]
I don't think this is a librosa issue, except insofar as we depend on llvmlite (through numba).

Since you're using a conda environment already, why not use the conda packages for llvmlite and numba (and conda-forge for librosa)?  Building your own will only cause more trouble, unless you have a strong reason to do so (eg developing llvmlite).

**Comment by rabitt** [comment]
I ran into the same issue recently - the issue is that llvmlite is not supported in Python 3.5
